### PR TITLE
doc: Restructure About the NCS page

### PR DESCRIPTION
**Pull request by richabp**
Restructure About the NCS page -
Change the title to Introduction
Combine the introductory content of Welcome & NCS page
Reduce the content of Git & West sections

Signed-off-by: Richa Pandey <richa.pandey@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/introduction.rst [hunk 1]
```diff
@@ -130,34 +111,16 @@ The |NCS| uses :ref:`west <zephyr:west>` to manage the combination of multiple G
...
-
-The ``west init`` command creates a west workspace, and you typically need to run it only once to initialize west with the revision of the |NCS| that you want to check out.
-It clones the manifest repository into the workspace.
-However, the content of the manifest repository is managed using Git commands, since west does not modify or update it.
-
-To clone the project repositories, use the ``west update`` command.
-This command makes sure your workspace contains Git repositories matching the projects defined in the manifest file.
-Whenever you check out a different revision in your manifest repository, you should run ``west update`` to make sure your workspace contains the project repositories the new revision expects (according to the manifest file).
-
-For more information about ``west init``, ``west update``, and other built-in commands, see :ref:`zephyr:west-built-in-cmds`.
```
Comment (FrancescoSer) [inline doc/nrf/introduction.rst:157]: I would keep the old line 159

For more information about the west tool, see the :ref:`zephyr:west` user guide.

**Review by FrancescoSer — CHANGES_REQUESTED** [review]

File: doc/nrf/introduction.rst (hunk 1 above)
Comment (richabp) [reply doc/nrf/introduction.rst:157]: Line 128 (or 109) also refers to the same user guide, hence removed due to repetition. But can be added if it seems to be relevant their as well.

File: doc/nrf/introduction.rst [hunk 2]
```diff
@@ -1,18 +1,25 @@
...
+It offers an extensible framework for building size-optimized software for memory-constrained devices as well as powerful and complex software for more advanced devices and applications.
 
-Every |NCS| release consists of a combination of all those repositories at different revisions.
-See the :ref:`repos_and_revs` section for a comprehensive list of repositories and their current revisions.
-The revision of each of those repositories is determined by the current revision of the main (manifest) repository, ``sdk-nrf``, which contains the SDK manifest file that helps manage the repositories as one :ref:`code base <dm_code_base>` with the :ref:`ncs_west_intro` tool.
+It integrates the Zephyr™ real-time operating system (RTOS) and a wide range of complete applications, samples, and protocol stacks such as Bluetooth® Low Energy, Bluetooth mesh, Matter, Thread/Zigbee and LTE-M/NB-IoT/GPS, TCP/IP.
+It also includes middleware such as CoAP, MQTT, LwM2M, various libraries, hardware drivers, Trusted Firmware-M for security, and a secure bootloader (MCUBoot).
```
Comment (FrancescoSer) [inline doc/nrf/introduction.rst:14]: MCUboot is actually not the only bootloader provided as there is also the NSIB.

The thing is that NSIB has a limited scope: to render the MCUboot image updatable via DFU. I wonder if we should mention NSIB somehow.

Comment (richabp) [reply doc/nrf/introduction.rst:14]: I believe this content is taken from here: https://academy.nordicsemi.com/topic/nrf-connect-sdk-structure-and-content/
where NSIB is not mentioned but we can consider mentioning it as well.

**Review by OisinFitz — APPROVED** [review]

**Review by FrancescoSer — APPROVED** [review]

**Pull request merged**